src/stores/useDashboard.ts

```diff
@@ -318,10 +318,31 @@ export const useDashboard = defineStore('dashboard', {
       if(window.location.hostname.search("testnet") > -1) {
         this.networkType = NetworkType.Testnet
       }
+
       const source: Record<string, LocalConfig> =
         this.networkType === NetworkType.Mainnet
-          ? import.meta.glob('../../chains/mainnet/*.json', { eager: true })
-          : import.meta.glob('../../chains/testnet/*.json', { eager: true });
+          ? import.meta.glob([
+            '../../chains/mainnet/chain4energy.json',
+            '../../chains/mainnet/chihuahua.json',
+            '../../chains/mainnet/cosmos.json',
+            '../../chains/mainnet/juno.json',
+            '../../chains/mainnet/lum.json',
+            '../../chains/mainnet/mars.json',
+            '../../chains/mainnet/osmosis.json',
+            '../../chains/mainnet/quicksilver.json',
+            '../../chains/mainnet/umee.json',
+          ], { eager: true })
+          : import.meta.glob([
+            '../../chains/testnet/chain4energy.json',
+            '../../chains/testnet/chihuahua.json',
+            '../../chains/testnet/cosmos.json',
+            '../../chains/testnet/juno.json',
+            '../../chains/testnet/lum.json',
+            '../../chains/testnet/mars.json',
+            '../../chains/testnet/osmosis.json',
+            '../../chains/testnet/quicksilver.json',
+            '../../chains/testnet/umee.json',
+          ], { eager: true });
```

Comment:
This is where the chains we display are set. Let me know if you want to add / remove any, or feel free to push a commit to update them. 

Doing it this way prevents new chains being displayed until we enable them. Without this change, any new json files that are added to the upstream branch will be displayed when we merge our fork with it.